recipes-omnect/omnect-device-service/omnect-health-check/omnect_service_log.tmpfilesd

```diff
@@ -0,0 +1,7 @@
+# Note:
+#   the log directory below can only be written by privileged prozesses, so
+#   any scripts wanting to do so need suitable permissions.
+#   as currently only omnect_service_log.sh is writing to this directory and
+#   only in the context of systemd hooks (PreStart/StopPost) it is sufficient
+#   to instruct systemd to let it run privileged.
+d /run/omnect_health_log 0775 root root -
```

Comment:
why habe it twice?
here and recipes-omnect/omnect-device-service/omnect-device-service/omnect-device-service.tmpfilesd

Reply:
this one was just forgotten to be removed after merging entries in omnect-device-service's tmpfiled file.